reference/webhooks.v3.yml

```
@@ -123,24 +156,207 @@ paths:
+              examples: {}
+        '400':
+          $ref: '#/components/responses/400_BadRequest'
+        '401':
+          $ref: '#/components/responses/401_Unauthorized'
+      tags:
+        - Webhooks Admin
+    put:
+      operationId: putHooksAdmin
+      summary: Upsert Email Notifications
+      description: |
+        Update email addresses to which notification emails will be sent when any domain associated with the API account is denylisted or when a webhook is deactivated. Supports `upsert` functionality in the case that no email address exists yet.
+      requestBody:
+        description: List of notification emails.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                emails:
+                  type: array
+                  items:
+                    type: string
+                    format: email
+                    example: 'webhooks@example.com'
+            examples:
+              Example:
+                value:
+                  emails:
+                    - user@example.com
+        required: true
+      responses:
+        '204':
+          description: No Content
+        '400':
+          $ref: '#/components/responses/400_BadRequest'
+        '401':
+          $ref: '#/components/responses/401_Unauthorized'
+        '422':
+          $ref: '#/components/responses/422_UnprocessableEntity'
+      tags:
+        - Webhooks Admin
+    parameters:
+      - schema:
+          type: string
+        name: store_hash
+        in: path
+        required: true
+        description: Hash of your BigCommerce store
+  '/stores/{store_hash}/v3/hooks/events':
+    get:
+      tags:
+        - Webhooks
+      summary: Get Events
+      parameters:
+        - $ref: '#/components/parameters/FilterPageParam'
+        - $ref: '#/components/parameters/FilterLimitParam'
+        - $ref: '#/components/parameters/FilterMaxCreatedAtParam'
+        - $ref: '#/components/parameters/FilterMinCreatedAtParam'
+      operationId: getWebhookEvents
+      responses:
+        '200':
+          description: Successful operation.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/HistoryEvent'
+                  meta:
+                    type: object
+                    properties:
+                      pagination:
+                        $ref: '#/components/schemas/Pagination'
+        '400':
+          $ref: '#/components/responses/400_BadRequest'
+        '401':
+          $ref: '#/components/responses/401_Unauthorized'
+      description: |
+        Get a list of events that were sent but not successfully received. Only events that are failed to be received are stored. Events are stored for not less than one week.

```

Sentence is both awkward and unnecessary.

```suggestion
        Get a list of events that were sent but not successfully received. Events are stored for not less than one week.
```